src/settings/StaffSlips/components/ViewSections/StaffSlipTemplateContentSection/StaffSlipTemplateContentSection.test.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  fireEvent,
+  within,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import {
+  Button,
+} from '@folio/stripes/components';
+import { PreviewModal } from '@folio/stripes-template-editor';
+// import buildStripes from '../../../../../../test/jest/__mock__/stripes.mock';
+
+import StaffSlipTemplateContentSection from './StaffSlipTemplateContentSection';
+
+const mockParseWithInstructionsReturnValue = 'testParsedValue';
+const mockTokensReducerReturnValue = 'testReturnValue';
+
+jest.mock('html-to-react', () => ({
+  __esModule: true,
+  default: {
+    ProcessNodeDefinitions: jest.fn(() => ({
+      processDefaultNode: 'testDefaultNode',
+    })),
+  },
+  Parser: () => ({
+    parseWithInstructions: jest.fn(() => mockParseWithInstructionsReturnValue),
+  }),
+}));
+jest.mock('@folio/stripes-template-editor', () => ({
+  PreviewModal: jest.fn(({
+    'data-testid': testId,
+  }) => (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+    <div data-testid={testId} />
+  )),
+  tokensReducer: () => mockTokensReducerReturnValue,
+}));
+
+Button.mockImplementation(jest.fn(({
+  children,
+  'data-testid': testId,
+  onClick,
+}) => (
+  // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+  <div
+    data-testid={testId}
+    onClick={onClick}
+  >
+    {children}
+  </div>
+)));
+
+describe('StaffSlipTemplateContentSection', () => {
+  const testIds = {
+    staffSlipsDisplayCol: 'staffSlipsDisplayCol',
+    staffSlipsPreviewButton: 'staffSlipsPreviewButton',
+    emailTemplateCol: 'emailTemplateCol',
+    previewModal: 'previewModal',
+  };
```

Comment:
Some of these ids are not used in the test. Could you check what we can do here?